src/mon/Monitor.h

```diff
@@ -558,6 +558,9 @@ class Monitor : public Dispatcher,
 public:
   epoch_t get_epoch();
   int get_leader() const { return leader; }
+  string get_leader_name() {
```

Comment:
This should be `get_leader_id()`.

While the MonMap considers each entry as the monitor's name, from the Monitor-class (or ceph-mon's) point of view, a `name` is in the form of `mon.<id>`.

And considering `MonMap::get_name()` will return the `id` part of the name, this could potentially lead to confusion in the future.

That said, either rename the function to  `get_leader_id()` or have the current function return in the form of `mon.<id>`.